boxoffice/views/order.py

```diff
@@ -438,6 +439,8 @@ def payment(order):
                 invoice_no=order.invoice_no,
             ),
         )
+        if app.config['TELEGRAM_APIKEY'] and app.config['TELEGRAM_CHATID']:
```

Comment:
This will fail if those keys are missing in config. Change to `if app.config.get('TELEGRAM_APIKEY') and app.config.get('TELEGRAM_CHATID')`